_articles/using-retries.md

```diff
@@ -0,0 +1,166 @@
+# Practical Circuit Breaking and Retries
+
+Circuit breaking and automatic retries are two powerful features of Envoy, and this article gives you a few examples of how to configure and use them.
+
+## The setup
+
+The Envoy documentation provides a good overview of
+[how to run the example](https://www.envoyproxy.io/docs/envoy/latest/start/sandboxes/zipkin_tracing)
+
+You should already have the following installed from running Envoy on your laptop:
+
+- [Docker](https://docs.docker.com/install/)
+- [Docker Compose](https://docs.docker.com/compose/install/)
+- [Git](https://help.github.com/articles/set-up-git/)
+- [curl](https://curl.haxx.se/)
+
+Additionally, install this service to drive load to our examples:
+
+- [wrk](https://github.com/wg/wrk)
+
+## Configuring latency and success rate
+
+In our first example we've add a configurable latency and success rate to the
+Python service that underlies all the Envoy examples.
+
+First, check out this tag of the example repo:
+
+`https://github.com/turbinelabs/envoy-examples/tree/step1`
```

Comment:
why is this URL in ticks? make it a link I think? or make it

```console
$ git clone https://github.com/turbinelabs/envoy-examples/tree/step1
```

Reply:
good question. fixing